index.html

```diff
@@ -2290,17 +2474,99 @@
+    <emu-alg><ol><li>Let <var>p</var> be ?&nbsp;<emu-xref aoid="ProxyCreate"><a href="https://tc39.es/ecma262/#sec-proxycreate">ProxyCreate</a></emu-xref>(<var>target</var>, <var>handler</var>).</li><li><ins>Let <var>signal</var> be ?&nbsp;<emu-xref aoid="GetProxySignal" id="_ref_0"><a href="#sec-get-proxy-signal">GetProxySignal</a></emu-xref>(<var>options</var>).</ins></li><li><ins>If <var>signal</var> is not <emu-val>undefined</emu-val>, then</ins><ol><li><ins>Let <var>p</var>.[[RevocationSignal]] be <var>signal</var>.[[HostRevocationSignal]].</ins></li></ol></li><li>Let <var>revokerClosure</var> be a new <emu-xref href="#sec-abstract-closure"><a href="https://tc39.es/ecma262/#sec-abstract-closure">Abstract Closure</a></emu-xref> with no parameters that captures nothing and performs the following steps when called:<ol><li>Let <var>F</var> be the <emu-xref href="#active-function-object"><a href="https://tc39.es/ecma262/#active-function-object">active function object</a></emu-xref>.</li><li>Let <var>p</var> be <var>F</var>.[[RevocableProxy]].</li><li>If <var>p</var> is <emu-val>null</emu-val>, return <emu-val>undefined</emu-val>.</li><li>Set <var>F</var>.[[RevocableProxy]] to <emu-val>null</emu-val>.</li><li><emu-xref href="#assert"><a href="https://tc39.es/ecma262/#assert">Assert</a></emu-xref>: <var>p</var> is a Proxy object.</li><li>Set <var>p</var>.[[ProxyTarget]] to <emu-val>null</emu-val>.</li><li>Set <var>p</var>.[[ProxyHandler]] to <emu-val>null</emu-val>.</li><li><ins>Set <var>p</var>.[[RevocationSignal]] to <emu-val>null</emu-val>.</ins></li><li>Return <emu-val>undefined</emu-val>.</li></ol></li><li>Let <var>revoker</var> be <emu-xref aoid="CreateBuiltinFunction"><a href="https://tc39.es/ecma262/#sec-createbuiltinfunction">CreateBuiltinFunction</a></emu-xref>(<var>revokerClosure</var>, 0, <emu-val>""</emu-val>, « [[RevocableProxy]] »).</li><li>Set <var>revoker</var>.[[RevocableProxy]] to <var>p</var>.</li><li>Let <var>result</var> be <emu-xref aoid="OrdinaryObjectCreate"><a href="https://tc39.es/ecma262/#sec-ordinaryobjectcreate">OrdinaryObjectCreate</a></emu-xref>(<emu-xref href="#sec-properties-of-the-object-prototype-object"><a href="https://tc39.es/ecma262/#sec-properties-of-the-object-prototype-object">%Object.prototype%</a></emu-xref>).</li><li>Perform !&nbsp;<emu-xref aoid="CreateDataPropertyOrThrow"><a href="https://tc39.es/ecma262/#sec-createdatapropertyorthrow">CreateDataPropertyOrThrow</a></emu-xref>(<var>result</var>, <emu-val>"proxy"</emu-val>, <var>p</var>).</li><li>Perform !&nbsp;<emu-xref aoid="CreateDataPropertyOrThrow"><a href="https://tc39.es/ecma262/#sec-createdatapropertyorthrow">CreateDataPropertyOrThrow</a></emu-xref>(<var>result</var>, <emu-val>"revoke"</emu-val>, <var>revoker</var>).</li><li>Return <var>result</var>.</li></ol></emu-alg>
```

Comment:
I'm not sure if you updated the spec text so step 4h, which sets the `[[RevocationSignal]]` to null despite step 3a.  

I'll bet it's standard for the spec editing code to put all of this on one line, but it's really hard on the eyes to have to scroll so far in to fix a bug on one line.

I will merge with this bug in, and I ask you to write a follow-up PR to fix the bug.

Per #6, I'd also like a follow-up to modify `new Proxy()` as well.

Reply:
Ah, I see it's generated from spec.emu below.  I didn't know that.